content/help/ops/traffic-management/deploy-guidelines/index.md

```diff
@@ -0,0 +1,205 @@
+spec:
+  hosts:
+  - "myapp.com" # or maybe "*" if you are testing without DNS using the ingress-gateway IP (e.g., http://1.2.3.4/hello)
+  gateways:
+  - myapp-gateway
+  http:
+  - match:
+    - uri:
+        prefix: /hello
+    route:
+    - destination:
+        host: helloworld.default.svc.cluster.local
+        subset: v1
+  - match:
+    ...
+{{< /text >}}
+
+Alternatively, you can combine both `VirtualServices` into one unit if possible:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: myapp
+spec:
+  hosts:
+  - myapp.com # cannot use "*" here since this is being combined with the mesh services
+  - helloworld.default.svc.cluster.local
+  gateways:
+  - mesh # applies internally as well as externally
+  - myapp-gateway
+  http:
+  - match:
+    - uri:
+        prefix: /hello
+      gateways:
+      - myapp-gateway #restricts this rule to apply only to ingress gateway
+    route:
+    - destination:
+        host: helloworld.default.svc.cluster.local
+        subset: v1
+  - match:
+    - gateways:
+      - mesh # applies to all services inside the mesh
+    route:
+    - destination:
+        host: helloworld.default.svc.cluster.local
+        subset: v1
+{{< /text >}}
+
+## Route rules have no effect on my application
+
+If route rules are working perfectly for the [Bookinfo](/docs/examples/bookinfo/) sample,
+but similar version routing rules have no effect on your own application, it may be that
+your Kubernetes services need to be changed slightly.
+
+Kubernetes services must adhere to certain restrictions in order to take advantage of
+Istio's L7 routing features.
+Refer to the [Requirements for Pods and Services](/docs/setup/kubernetes/spec-requirements)
+for details.
+
+## Envoy won't connect to my HTTP/1.0 service
+
+Envoy requires HTTP/1.1 or HTTP/2 traffic for upstream services. For example, when using [NGINX](https://www.nginx.com/) for serving traffic behind Envoy, you
+will need to set the [proxy_http_version](https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_http_version) directive in your NGINX configuration to be "1.1", since the NGINX default is 1.0
+
+Example configuration:
+
+{{< text plain >}}
+upstream http_backend {
+    server 127.0.0.1:8080;
+
+    keepalive 16;
+}
+
+server {
+    ...
+
+    location /http/ {
+        proxy_pass http://http_backend;
+        proxy_http_version 1.1;
+        proxy_set_header Connection "";
+        ...
+    }
+}
+{{< /text >}}
+
+## Headless TCP services losing connection from Istiofied containers
+
+If `istio-citadel` is deployed, Envoy is restarted every 15 minutes to refresh certificates.
+This causes the disconnection of TCP streams or long-running connections between services.
+
+You should build resilience into your application for this type of
+disconnect, but if you still want to prevent the disconnects from
+happening, you will need to disable mutual TLS and the `istio-citadel` deployment.
```

Comment:
I think you still need istio-citadel even with Mutual TLS is disabled...  @myidpt comments?